docs/sources/configure/_index.md

```diff
@@ -880,6 +880,40 @@ volume_results_cache:
   # compression. Supported values are: 'snappy' and ''.
   # CLI flag: -frontend.volume-results-cache.compression
   [compression: <string> | default = ""]
+
+# Cache series query results.
+# CLI flag: -frontend.cache-series-results
```

Comment:
other result caches use `-querier.` prefix, but ideally all of them should use `-frontend.` I think.
note: these two new flags `-frontend.cache-series-results`, `-frontend.cache-label-results` use inconsistent naming when compared to the existing ones

Reply:
Good catch.
I think currently it's mix of both. For boolean (to decide whether to cache) we use `querier.` prefix. For actual cache config we use `frontend.` prefix.

e.g: `-querier.cache-index-stats-results` but `frontend.index-stats-results-cache`

We can also do same for two new flags. I will fix it.

Reply:
pushed.